src/sentry/digests/utilities.py

```diff
@@ -23,6 +24,13 @@ def get_digest_metadata(digest):
     return start, end, counts
 
 
+def should_get_personalized_digests(target_type: ActionTargetType, project_id: int) -> bool:
```

Comment:
This looks like a new function - is the logic copied from elsewhere I can't find? I wanna be super extra sure this is correct. I think it's from the old `get_personalized_digests`, is that right?

Reply:
https://github.com/getsentry/sentry/blob/master/src/sentry/digests/utilities.py#L36-L39
```python
def get_personalized_digests(target_type, project_id, digest, user_ids):
    if (
        target_type == ActionTargetType.ISSUE_OWNERS
        and ProjectOwnership.objects.filter(project_id=project_id).exists()
    ):
	    ...